mscore/pianoroll/pianoview.cpp

```diff
@@ -1953,10 +1953,10 @@ void PianoView::finishNoteGroupDrag() {
 //   pasteNotes
 //---------------------------------------------------------
 
-void PianoView::pasteNotes(const QString& data, Fraction pasteStartTick, int pitchOffset, bool xIsOffset)
+void PianoView::pasteNotes(const QString& dat, Fraction pasteStartTick, int pitchOffset, bool xIsOffset)
```

Comment:
One more `dat` that could be renamed to `copiedNotes`.

BTW, I suggest that you also rename the parameter in the declaration of this function in the header file.

Reply:
done